examples/newsreaderExample/src/main/java/io/realm/examples/newsreader/ui/details/DetailsPresenter.java

```diff
@@ -51,32 +49,24 @@ public void onCreate() {
     @Override
     public void onResume() {
         // Show story details
-        Subscription detailsSubscription = model.getStory(storyId)
-                .subscribe(new Action1<NYTimesStory>() {
-                    @Override
-                    public void call(NYTimesStory story) {
-                        view.hideLoader();
-                        view.showStory(story);
-                        view.setRead(story.isRead());
-                    }
+        Disposable detailsSubscription = model.getStory(storyId)
+                .subscribe(story -> {
+                    view.hideLoader();
+                    view.showStory(story);
+                    view.setRead(story.isRead());
                 });
 
         // Mark story as read if screen is visible for 2 seconds
-        Subscription timerSubscription = Observable.timer(2, TimeUnit.SECONDS)
+        Disposable timerSubscription = Observable.timer(2, TimeUnit.SECONDS)
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(new Action1<Long>() {
-                    @Override
-                    public void call(Long aLong) {
-                        model.markAsRead(storyId, true);
-                    }
-                });
-        
-        subscriptions = new CompositeSubscription(detailsSubscription, timerSubscription);
+                .subscribe(aLong -> model.markAsRead(storyId, true));
+
+        disposables = new CompositeDisposable(detailsSubscription, timerSubscription);
     }
 
     @Override
     public void onPause() {
-        subscriptions.unsubscribe();
+        disposables.dispose();
```

Comment:
Most people seem to think that `disposables.clear()` is the better to use: https://blog.kaush.co/2017/06/21/rxjava-1-rxjava-2-disposing-subscriptions/

Same in other places

Reply:
If you do `disposables = null` afterwards and re-create it when it is null then `dispose()` also works well.

`clear()` means you can re-use it, but you don't have to if you don't want to

Reply:
I will try to reuse disposables.

Reply:
I dunno, I tend to just use `dispose()` and be like whatever, it really depends on if you want to re-use them. For example, in Fragments, it might be safer to use `clear()` unless you do re-create the composite disposable.